sbol_utilities/helper_functions.py

```diff
@@ -363,4 +363,18 @@ def is_circular(obj: Union[sbol3.Component, sbol3.LocalSubComponent, sbol3.Exter
     :param obj: design to be checked
     :return: true if circular
     """    
-    return any(n==sbol3.SO_CIRCULAR for n in obj.types)
\ No newline at end of file
+    return any(n==sbol3.SO_CIRCULAR for n in obj.types)
```

Comment:
Per standard Python style, there should be two blank lines before the new function.